setup.py

```diff
@@ -20,6 +20,9 @@
 SETUP_DIR = os.path.dirname(__file__)
 README = os.path.join(SETUP_DIR, 'README.md')
 
+version = '0.0.1'
```

Comment:
```suggestion
version = '0.1.0'
```
Shall we make this the client `0.1.0` and keep increasing that minor as we add features?